frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -346,46 +349,52 @@ class TagEditorParamInner extends Component<Props> {
           </InputContainer>
         )}
 
-        <InputContainer lessBottomPadding>
-          <ContainerLabel>{t`Required?`}</ContainerLabel>
-          <Toggle
-            id="tag-editor-required"
-            value={tag.required}
-            onChange={value => this.setRequired(value)}
+        <div>
```

Comment:
@nemanjaglumac now that it's wrapped in a `div`, not a label, the bug with automatic unchecking the "required" switch is fixed.